judo-ui-react/src/main/resources/actor/src/components/widgets/AggregationInput.tsx.hbs

```diff
@@ -180,22 +173,23 @@ export const AggregationInput = ({
                       })}
                     >
                       {loading ? <CircularProgress color="inherit" size='1rem' className="AggregationInputLoading" /> : null}
-                      {onSet && !exists(value) && !readOnly && (
-                        <IconButton className={`${name}-set`} disabled={disabled} onClick={ (event: any) => { handleDropdownClose(event); onSet(); } } title={t('judo.component.AggregationInput.open-filter', { defaultValue: 'Open selector dialog' }) as string}>
-                          <MdiIcon path="magnify" />
-                        </IconButton>
-                      )}
-                      {exists(value) && onView && (
-                        <IconButton className={`${name}-view`} disabled={editMode && !isInlineCreatable} onClick={ (event: any) => { handleDropdownClose(event); onView(); } } title={t('judo.component.AggregationInput.navigate', { defaultValue: 'Navigate to element' }) as string}>
-                          <MdiIcon path="eye" />
-                        </IconButton>
-                      )}
-                      {!exists(value) && onCreate && !readOnly && (
-                        <IconButton className={`${name}-create`} disabled={disabled || (editMode && !isInlineCreatable)} onClick={ (event: any) => { handleDropdownClose(event); onCreate(); } } title={t('judo.component.AggregationInput.create', { defaultValue: 'Create' }) as string}>
-                          <MdiIcon path="file_document_plus" />
-                        </IconButton>
-                      )}
-                      {exists(value) && (onEdit || onDelete || onRemove || onSet || onUnset) && (
+                      { aggregationInputButtonProps.filter(a => a.name === 'set' || a.name === 'view' || a.name === 'create').map( (aggregationInputButtonProp: AggregationInputButtonProps) => 
+                          aggregationInputButtonProp.visible && aggregationInputButtonProp.visible(value, aggregationInputButtonProp.onClick) && (
+                            <IconButton
+                              className={aggregationInputButtonProp.className}
```

Comment:
We are not setting the `key` prop which I assume we can see console errors for.

Reply:
[141d4bf](https://github.com/BlackBeltTechnology/judo-ui-react-template/pull/378/commits/141d4bf18275f59a778c628641bce06e4b946e7f)